modulus/datapipes/gnn/stokes_dataset.py

```diff
@@ -0,0 +1,306 @@
+# Copyright (c) 2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import re
+from typing import List, Tuple, Dict, Union, Optional, Any
+
+import numpy as np
+import torch
+from torch.nn import functional as F
+
+from .utils import read_vtp_file, save_json, load_json
+
+try:
+    import dgl
+    from dgl.data import DGLDataset
+except:
+    raise ImportError(
+        "Stokes flow Dataset requires the DGL library. Install the "
+        + "desired CUDA version at: \n https://www.dgl.ai/pages/start.html"
+    )
+
+try:
+    import vtk
+    import pyvista as pv
+except:
+    raise ImportError(
+        "Stokes flow Dataset requires the vtk and pyvista libraries. Install with "
+        + "pip install vtk pyvista"
+    )
+
+
+class StokesDataset(DGLDataset):
+    """In-memory Stokes flow Dataset"""
```

Comment:
Can you add more docs here? Especially information on the parameters would be nice. Please refer: https://github.com/NVIDIA/modulus/blob/main/modulus/datapipes/gnn/ahmed_body_dataset.py#L62

Reply:
Thank you for this comments! I have added detailed descritpion for this dataset.